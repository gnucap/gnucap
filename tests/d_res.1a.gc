spice

.param r=0

* bug: r2 has a common, r1 doesn't
r1 1 0 {r}
r2 1 2 r

r0 3 2 1m
v1 3 0 1

.print dc v(nodes)
.dc trace=i
.print dc v(nodes) y(r*)
.dc trace=i

.status notime
.end
