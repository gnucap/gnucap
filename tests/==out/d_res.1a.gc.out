#           v(1)       v(2)       v(3)      
r1: short circuit
 0.         0.         0.         0.        
r2: short circuit
-1.         0.0097087  0.019417   0.99029   
r2: short circuit
-2.         0.009803   0.0098973  0.9902    
r2: short circuit
-3.         0.0098039  0.0098048  0.9902    
r2: short circuit
-4.         0.0098039  0.0098039  0.9902    
r2: short circuit
 0.         0.0098039  0.0098039  0.9902    
#           v(1)       v(2)       v(3)       y(r1)      y(r2)      y(r0)     
r1: short circuit
 0.         0.         0.         0.         100.K      100.K      1.K       
r1: short circuit
r2: short circuit
-1.         0.0097087  0.019417   0.99029    100.K      100.K      1.K       
r1: short circuit
r2: short circuit
-2.         0.009803   0.0098973  0.9902     100.K      100.K      1.K       
r1: short circuit
r2: short circuit
-3.         0.0098039  0.0098048  0.9902     100.K      100.K      1.K       
r1: short circuit
r2: short circuit
-4.         0.0098039  0.0098039  0.9902     100.K      100.K      1.K       
r1: short circuit
r2: short circuit
 0.         0.0098039  0.0098039  0.9902     100.K      100.K      1.K       
Gnucap   System status
iterations: op=0, dc=5, tran=0, fourier=0, total=10
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=3, subckt=0, model=0, total=3
dctran density=77.8%, ac density=77.8%
